package.json

```diff
@@ -81,5 +83,8 @@
   },
   "engines": {
     "node": ">=6.0.0"
+  },
+  "bin": {
+    "postinstall": "touch ./src/Frontend/config/recaptcha.js"
```

Comment:
not sure if you got what I wanted to say:

a postinstall script is fine, but the logic in there could be moved into a shell file (e.g. `generate-frontend-recaptcha.sh` placed in the `bin/` dir in the project's root).
Furthermore this file should check if a config file already exists before creating one (`if [ -e 'path/to/config' ]; `).

And if a new one is created, something like this could be put into the file from shell: `export default ''` (the user should add the key here too if the registration is needed).